docs/sphinx/conf.py

```diff
@@ -31,7 +31,7 @@
 read_the_docs_build = os.environ.get('READTHEDOCS', None) == 'True'
 
 if read_the_docs_build:
-    print 'Running on read the docs'
```

Comment:
@davidbeckingsale and @kab163, readthedocs was giving a build error here (possibly running newer version of python?).  So, I added parenthesis and it started working again.